tests/test_connectors/test_ssh.py

```diff
@@ -540,7 +540,7 @@ def test_put_file_su_user_fail(self, fake_sftp_client, fake_ssh_client):
         assert status is False
 
         fake_ssh_client().exec_command.assert_called_with((
-            "su centos -c 'sh -c '\"'\"'mv "
+            "su centos -c 'sh -c '\"'\"'cp "
```

Comment:
This command will now be the `acl` one I believe - as the mock above is failing the first shell command executed. Changing this to assert for the acl command should cover the test.

Would also be good to have an addition test for the `cp` command failing vs. first ACL one, but I'm happy to add that in as well.